core/cloudflow-blueprint/src/test/scala/cloudflow/blueprint/BlueprintSpec.scala

```diff
@@ -252,7 +252,10 @@ class BlueprintSpec extends WordSpec with MustMatchers with EitherValues with Op
       blueprint.problems.collect { case unconnected: UnconnectedInlets ⇒ unconnected }.size mustBe 0
       val paths = Vector(VerifiedPortPath(ingressRef.name, "out"), VerifiedPortPath(egress2Ref.name, "in"))
         .sortBy(_.toString)
-      blueprint.problems mustBe Vector(
+      blueprint.problems.filterNot {
```

Comment:
this `filterNot`...`true` is so confusing.

Reply:
flipped it for you.